clients/admin-ui/src/features/privacy-requests/events-and-logs/ActivityTimeline.tsx

```diff
@@ -1,20 +1,14 @@
-import {
-  Box,
-  ErrorWarningIcon,
-  Flex,
-  GreenCheckCircleIcon,
-  Text,
-} from "fidesui";
+import { Box, Text, useDisclosure } from "fidesui";
 import { ExecutionLog, PrivacyRequestEntity } from "privacy-requests/types";
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 import { ExecutionLogStatus } from "~/types/api";
 
-import { EventData } from "./EventDetails";
+import LogDrawer from "./LogDrawer";
+import TimelineEntry from "./TimelineEntry";
 
 type ActivityTimelineProps = {
   subjectRequest: PrivacyRequestEntity;
-  setEventDetails: (d: EventData) => void;
 };
 
 const hasUnresolvedError = (entries: ExecutionLog[]) => {
```

Comment:
I know this method existed before- but mind adding a short docstring explaining what this does? Specifically- what an "unresolved" error means vs just an error?